src/Summary.jsx

```diff
@@ -29,18 +29,16 @@ class Summary extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeArea: {},
       connError: false,
       layers: {},
     };
   }
 
   componentDidMount() {
     const { activeArea } = this.props;
-    const blockId = (activeArea && activeArea.id) ? activeArea.id : 'LLA 0970';
+    const blockId = (activeArea && activeArea.id) ? activeArea.id : null;
```

Comment:
With @cazapatamar talked about what to do when activeArea is not set, because in this case you are still making a request to the backend, and the idea is no to do it